tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -352,6 +355,88 @@ def test_5_inc_snapshots(network_config,
     test_matrix.run_test(_test_seq_snapshots)
 
 
+DEFAULT_HOST_IP = "192.168.0.1"
+DEFAULT_GUEST_IP = "192.168.0.2"
+DEFAULT_TAP_NAME = "tap0"
+DEFAULT_NETMASK = 30
+
+
+def _get_process_start_time(p_pid):
+    # Helper function returning:
+    # - start time of process with provided PID
+    # - return code of stat -c%X /proc/$$ command
+    proc = subprocess.Popen(["stat -c%X /proc/{}".format(p_pid)],
+                            stdout=subprocess.PIPE,
+                            shell=True,
+                            encoding='ascii')
+    out = proc.communicate()[0].strip('\n')
+    return out, proc.returncode
+
+
+@pytest.mark.skipif(
+    platform.machine() != "x86_64",
+    reason="Not supported yet."
+)
+def test_load_snapshot_failure_handling(bin_cloner_path):
+    """
+    Test scenario.
+
+    1. Create two empty files representing snapshot memory and
+    microvm state
+    2. Try to load a VM snapshot out of the empty files.
+    3. Verify that an error was shown and the FC process is terminated.
+    """
+    logger = logging.getLogger("snapshot_load_failure")
+    vm = MicrovmBuilder(bin_cloner_path).create_basevm()
+    vm.spawn(log_level='Info')
+
+    vm.create_tap_and_ssh_config(host_ip=DEFAULT_HOST_IP,
+                                 guest_ip=DEFAULT_GUEST_IP,
+                                 netmask_len=DEFAULT_NETMASK,
+                                 tapname=DEFAULT_TAP_NAME)
```

Comment:
nit: from what I can tell, you don't need a TAP or configuring microVM networking for this test
```suggestion
```

Reply:
You are correct